openai/openai.go

```diff
@@ -585,6 +590,7 @@ type ChatWriter struct {
 	stream        bool
 	streamOptions *StreamOptions
 	id            string
+	toolCallsSent bool
```

Comment:
```suggestion
	toolCallSent bool
```

Reply:
just a small change on the naming then we're good to go @anuraaga

Reply:
I think I already applied this change, can you double-check?

Reply:
Oops, realized hadn't changed the function parameter to match, fixed it